apps/web/src/components/auth/LoginForm.tsx

```diff
@@ -24,6 +24,10 @@ export function LoginForm({}: Props) {
       password: string;
     }
   >((data) => api.post(`/v1/auth/login`, data));
+  const [params] = useSearchParams();
+  const code = params.get('code');
+  const next = params.get('next');
+  const isFromVercel = code && next;
```

Comment:
This check should be part of `apps/web/src/api/hooks/use-vercel-integration.ts` instead because it is used multiple times across the codebase (DRY-Principle)